Tools-Override/depProj.targets

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+***********************************************************************************************
+depProj.targets
```

Comment:
What changed on this file? Usually when we add a file to Tools-Override we separate it in two commits so that the first would add the file unmodified and the second commit shows the changes made on the file.